mobile/test/common/integration/test_server.h

```diff
@@ -38,7 +38,7 @@ class TestServer {
    * Starts the server. Can only have one server active per JVM. This is blocking until the port can
    * start accepting requests.
    */
-  void startTestServer(bool use_quic);
+  void startTestServer(bool use_quic, bool disable_https);
```

Comment:
nit: please comment to explain what http version is implied by these arguments

Reply:
Please explain that when use_quic is true, disable_https must be false. It's too bad we can't use enum, as that would be so much cleaner :(